components/HorizontalCardListFeature/HorizontalCardListFeature.js

```diff
@@ -84,10 +90,12 @@ function HorizontalCardListFeature(props = {}) {
       </Box>
     );
   }
+
+  // TEST PRAYER CARDS!
```

Comment:
@leoalfonso14 Did you mean to leave this comment? Do we still need to test?

Reply:
Yes, I do not have prayer cards and I thought you guys did, so when testing check that those look good. I can create some and test them, but I thought you guys already had some created.

Reply:
Okay!